config/rbac/operator-role.yaml

```diff
@@ -71,6 +71,7 @@ rules:
   - configmaps
   - secrets
   - serviceaccounts
+  - namespaces
```

Comment:
It doesn't seem like a best practice to grant all verbs on `namespaces` resources, especially `delete` and `deletecollection`.  Could this be refined to the minimal set of verbs?